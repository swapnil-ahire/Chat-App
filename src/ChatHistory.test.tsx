import React from "react";
import { shallow } from 'enzyme';
import ChatHistory from './ChatHistory';

describe('ChatHistory', () => {
    it('Renders ChatHistory', () => {
        const chatHistory = shallow(<ChatHistory />);
        expect(chatHistory).toMatchSnapshot();
    });

    it('Renders ChatHistory with classname', () => {
        const chatHistory = shallow(<ChatHistory className="my-class" />);
        expect(chatHistory).toMatchSnapshot();
    });
});