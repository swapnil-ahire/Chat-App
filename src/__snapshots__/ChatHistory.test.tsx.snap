// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatInterface Renders ChatInterface 1`] = `ShallowWrapper {}`;

exports[`ChatInterface Renders ChatInterface with classname 1`] = `ShallowWrapper {}`;
